.travis.yml

```diff
@@ -11,9 +11,9 @@ cache:
 before_script:
   - cp .env.travis .env
   - mysql -e 'create database `header-checker-rest`;'
-  - composer self-update
```

Comment:
Why removing this? Of course we would like Composer to stay up-to-date, to make sure its own bugs are fixed.

Reply:
Composer is always up-to-date in Travis. 👍🏻